addon/data/error-panel.js

```diff
@@ -23,6 +23,9 @@ self.port.on("showError", function (error) {
   if (error.stack) {
     console.error(error.stack);
   }
+  unsafeWindow.error = cloneInto(error, unsafeWindow);
+  unsafeWindow.captureException(unsafeWindow.error);
+
```

Comment:
I decided not to, because cloneInto fixes the weird problems. It does a structured clone into the new window, so there are no proxies or weird wrappers.